src/types/runtime.go

```diff
@@ -47,8 +49,9 @@ type ZarfFindImagesOptions struct {
 
 // ZarfDeployOptions tracks the user-defined preferences during a package deploy.
 type ZarfDeployOptions struct {
-	AdoptExistingResources bool `json:"adoptExistingResources" jsonschema:"description=Whether to adopt any pre-existing K8s resources into the Helm charts managed by Zarf"`
-	SkipWebhooks           bool `json:"componentWebhooks" jsonschema:"description=Skip waiting for external webhooks to execute as each package component is deployed"`
+	AdoptExistingResources bool          `json:"adoptExistingResources" jsonschema:"description=Whether to adopt any pre-existing K8s resources into the Helm charts managed by Zarf"`
+	SkipWebhooks           bool          `json:"componentWebhooks" jsonschema:"description=Skip waiting for external webhooks to execute as each package component is deployed"`
+	Timeout                time.Duration `json:"timeout" jsonschema:"description=Timeout for performing Helm operations"`
```

Comment:
Since this is a `time.Duration` and not `string` of seconds, it might be good to show some examples

Reply:
It actually does when you run help since the default shows.  We could show more but I was copying Helm here: https://deploy-preview-2157--zarf-docs.netlify.app/docs/the-zarf-cli/cli-commands/zarf_package_deploy#options